doc/library-reference/drivers/sensors/dht.rst

```diff
@@ -4,7 +4,23 @@
 .. module:: dht
    :synopsis: DHT temperature and humidity sensor.
 
-----------------------------------------------
+DHT is temperature and humidity sensor. Two most
+popular versions are DHT11 (blue) and DHT22 (white).
+
+.. image:: ../../../images/drivers/DHT.png
+   :width: 50%
+   :target: ../../../_images/DHT.png
+
+They are available in bare sensor or module variants.
```

Comment:
I'm not very good in writing descriptions. So feel free to replace it with your own words.

Reply:
Ok, I'll have a look at it.